recipes/plotly/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "1.12.4" %}
+{% set sha256 = "6136ebcc6cd6435b5d238385b395d856e75773a58c303c5107c9834ca5577bbd" %}
+{% set md5 = "f1ff6b1b18554258aab50449ab571ee2" %}
+
+package:
+  name: plotly
+  version: {{ version }}
+
+source:
+  fn: plotly-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/plotly/plotly-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - requests
+    - pytz
+    - six
+
+test:
+  imports:
+    - plotly
+  commands:
+    - python -c "import plotly; print(plotly.version.__version__)"
```

Comment:
This is essentially the same as the line above. This is what the `imports` block does (minus printing the version).